<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormularioUsuario</class>
 <widget class="QDialog" name="FormularioUsuario">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>520</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>476</width>
    <height>520</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>476</width>
    <height>520</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/prefijoNuevo/logo.png</normaloff>:/prefijoNuevo/logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="1" column="0">
       <widget class="QFrame" name="frame_3">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#frame_3{
background-color: rgb(255, 255, 255);
border-Bottom-left-radius: 10px;
border-Bottom-right-radius: 10px;
}

QLabel{
font-family: Roboto;
font-style: normal;
font-weight: 450;
font-size: 15px;
letter-spacing: 0.02em;
color: #6B7280;}


QLineEdit{
font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
letter-spacing: 0.02em;
background: #F3F4F6;
border: 2px solid #F3F4F6;
border-radius: 8px;
color:#9CA3AF;
padding:10px}

QLineEdit::focus{
background: #FFFFFF;
border: 2px solid #3A4F50;
border-radius: 8px;
}


/* DATE */
QDateEdit
{
font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
letter-spacing: 0.02em;
background: #F3F4F6;
color:#9CA3AF;
padding:8px;
border-radius:8px;
}

QDateEdit::drop-down
{
	image: url(:/prefijoNuevo/025-calendar-13.png);
    width:15px;
    height:15px;
padding:13px 20px 10px 10px;
}

QDateEdit::focus{
background: #FFFFFF;
border: 2px solid #3A4F50;
border-radius: 8px;
}

QComboBox{
font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
letter-spacing: 0.02em;
background: #F3F4F6;
color:#9CA3AF;
border-radius:8px;
padding:10px;
combobox-popup: 0;
border: 2px solid #F3F4F6;

}


QComboBox::drop-down
{
    width: 20px;
    border-left-width: 1px;
	padding: 10px 10px 10px 20px;
 }


QComboBox QAbstractItemView {
	color:#9CA3AF;
	font-size:14px;
	background-color: #F3F4F6;
	selection-background-color:#344647;
	selection-color:#ffffff;
	outline: 0px;
   border: 1px solid #344647;
 border-radius:8px;
padding:10px
}



QComboBox::down-arrow
{
	image: url(:/menu/contraerabajo.png);
     width: 11px;
     height: 11px;
}

QComboBox::focus{
background: #FFFFFF;
border: 2px solid #344647;
border-radius: 8px;
}

QComboBox {combobox-popup: 0}

QRadioButton{font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
color:#9CA3AF}






</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QPushButton" name="btn_guardar">
         <property name="geometry">
          <rect>
           <x>320</x>
           <y>426</y>
           <width>136</width>
           <height>40</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>136</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>40</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    font-family: Roboto;
    font-style: normal;
    font-weight: normal;
    font-size: 15px;
    letter-spacing: 0.02em;
    color: white;
    padding: 8px 16px;
    background: #3f5758;
    border: none; 
    border-radius: 8px; 
}

QPushButton:hover {
    background: #4a6b6c; 
    border: none;
    border-radius: 8px;
}


QPushButton:pressed {
    background: #2e4546;
    border: none;
    border-radius: 8px;
}</string>
         </property>
         <property name="text">
          <string>Guardar</string>
         </property>
         <property name="iconSize">
          <size>
           <width>15</width>
           <height>21</height>
          </size>
         </property>
        </widget>
        <widget class="QFrame" name="frame_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>461</width>
           <height>411</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">/* QLabel para Mostrar Error */
QLabel#errorNombres, #errorApellidos, #errorIdentificacion, #errorFechaNacimiento, #errorCorreo, #errorTelefono, #errorDireccion, #errorContrasena{
    font-family: Roboto;
    font-style: normal;
    font-weight: bold;
    font-size: 14px;
    color: #D32F2F; 
}
</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="lbl5">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>113</y>
            <width>151</width>
            <height>21</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Apellidos*</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
          </property>
         </widget>
         <widget class="QLineEdit" name="line_dni">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>36</y>
            <width>205</width>
            <height>40</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="maxLength">
           <number>11</number>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="placeholderText">
           <string>Ejm:  9929289833</string>
          </property>
         </widget>
         <widget class="QLabel" name="lbl11">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>171</width>
            <height>21</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>CÃ©dula*</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
          </property>
         </widget>
         <widget class="QLabel" name="lbl9">
          <property name="geometry">
           <rect>
            <x>240</x>
            <y>10</y>
            <width>81</width>
            <height>21</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Nombre*</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
          </property>
         </widget>
         <widget class="QLineEdit" name="line_nombre">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>240</x>
            <y>36</y>
            <width>205</width>
            <height>40</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="maxLength">
           <number>100</number>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="placeholderText">
           <string>Ingrese nombre</string>
          </property>
          <property name="clearButtonEnabled">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QLineEdit" name="line_apeP">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>138</y>
            <width>205</width>
            <height>40</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="maxLength">
           <number>100</number>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="placeholderText">
           <string>Ingrese apellidos</string>
          </property>
         </widget>
         <widget class="QLabel" name="lbl10">
          <property name="geometry">
           <rect>
            <x>320</x>
            <y>212</y>
            <width>82</width>
            <height>21</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Cargo*</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
          </property>
         </widget>
         <widget class="QLabel" name="lbl3">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>212</y>
            <width>151</width>
            <height>21</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Correo electronico*</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
          </property>
         </widget>
         <widget class="QLineEdit" name="line_correo">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>236</y>
            <width>291</width>
            <height>40</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="maxLength">
           <number>100</number>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="placeholderText">
           <string>Ingrese correo</string>
          </property>
         </widget>
         <widget class="QLabel" name="lbl6_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>310</y>
            <width>151</width>
            <height>21</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>ContraseÃ±a*</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
          </property>
         </widget>
         <widget class="QLabel" name="lbl6_3">
          <property name="geometry">
           <rect>
            <x>242</x>
            <y>110</y>
            <width>81</width>
            <height>21</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Estado*</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
          </property>
         </widget>
         <widget class="QComboBox" name="cbox_estado">
          <property name="geometry">
           <rect>
            <x>240</x>
            <y>140</y>
            <width>205</width>
            <height>40</height>
           </rect>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="currentIndex">
           <number>1</number>
          </property>
          <item>
           <property name="text">
            <string>Inactivo</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Activo</string>
           </property>
          </item>
         </widget>
         <widget class="QRadioButton" name="admi">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>320</x>
            <y>238</y>
            <width>131</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Administrador</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QRadioButton" name="tutor">
          <property name="geometry">
           <rect>
            <x>320</x>
            <y>260</y>
            <width>111</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Otros</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="line_contrasena">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>340</y>
            <width>291</width>
            <height>38</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>Ingrese contraseÃ±a</string>
          </property>
         </widget>
         <widget class="QLabel" name="errorNombres">
          <property name="geometry">
           <rect>
            <x>242</x>
            <y>77</y>
            <width>201</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="errorApellidos">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>179</y>
            <width>201</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="errorIdentificacion">
          <property name="geometry">
           <rect>
            <x>12</x>
            <y>76</y>
            <width>201</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="errorCorreo">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>277</y>
            <width>291</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="errorContrasena">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>380</y>
            <width>291</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QFrame" name="frame_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#frame_2{background: #3A4F50 ;border-top-left-radius: 10px;
border-top-right-radius: 10px;}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QToolButton" name="btn_closeedit">
         <property name="geometry">
          <rect>
           <x>450</x>
           <y>6</y>
           <width>25</width>
           <height>25</height>
          </rect>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="toolTip">
          <string>Cerrar</string>
         </property>
         <property name="styleSheet">
          <string notr="true">#btn_closeedit{
font-family: Roboto;
font-weight: normal;
font-size: 14px;
background-color: #d5d5d8;
color: #6B7280;
border-radius:11px
}



#btn_closeedit:hover{background:#ce4d3a ;
border-radius:11px; }

</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../image/source.qrc">
           <normaloff>:/menu/x.png</normaloff>:/menu/x.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>13</width>
           <height>13</height>
          </size>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="lbl_titulo_ventana">
         <property name="geometry">
          <rect>
           <x>19</x>
           <y>4</y>
           <width>241</width>
           <height>19</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 18px;
line-height: 23px;
letter-spacing: 0.02em;
color: #FFFFFF;}</string>
         </property>
         <property name="text">
          <string>Nuevo Usuario</string>
         </property>
        </widget>
        <widget class="QLabel" name="lbl_usuariologeado">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>27</y>
           <width>341</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
line-height: 23px;
letter-spacing: 0.02em;
color: #FFFFFF;}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>line_dni</tabstop>
  <tabstop>line_nombre</tabstop>
  <tabstop>line_apeP</tabstop>
  <tabstop>cbox_estado</tabstop>
  <tabstop>line_correo</tabstop>
  <tabstop>admi</tabstop>
  <tabstop>tutor</tabstop>
 </tabstops>
 <resources>
  <include location="../image/source.qrc"/>
 </resources>
 <connections/>
</ui>
